client/my-sites/checkout/checkout-thank-you/jetpack-checkout-siteless-thank-you-completed.tsx

```diff
@@ -69,39 +94,86 @@ const JetpackCheckoutSitelessThankYouCompleted: FC< Props > = ( {
 								: 'jetpack-checkout-siteless-thank-you-completed__main-message'
 						}
 					>
-						{ translate( 'Your %(productName)s subscription will be activated soon', {
-							args: {
-								productName,
-							},
-						} ) }
+						{ title }
 					</h1>
-					<p>
-						{ translate(
-							'As soon as your subscription is activated you will receive a confirmation email from our Happiness Engineers.'
-						) }
-					</p>
-					<p>
-						{ translate(
-							'If you prefer to setup Jetpack with the help of our Happiness Engineers, {{a}}schedule a 15 minute call now{{/a}}.',
-							{
-								components: {
-									a: (
-										<a
-											className="jetpack-checkout-siteless-thank-you-completed__link"
-											onClick={ () =>
-												dispatch(
-													recordTracksEvent( 'calypso_siteless_checkout_happiness_link_clicked', {
-														product_slug: productSlug,
-													} )
-												)
-											}
-											href={ happinessAppointmentLink }
-										/>
-									),
-								},
-							}
-						) }
-					</p>
+					{ ! automaticTransferSucceeded && (
+						<>
+							<p>
+								{ translate(
+									'As soon as your subscription is activated you will receive a confirmation email from our Happiness Engineers.'
+								) }
+							</p>
+							<p>
+								{ translate(
+									'If you prefer to setup Jetpack with the help of our Happiness Engineers, {{a}}schedule a 15 minute call now{{/a}}.',
+									{
+										components: {
+											a: (
+												<a
+													className="jetpack-checkout-siteless-thank-you-completed__link"
+													onClick={ () =>
+														dispatch(
+															recordTracksEvent(
+																'calypso_siteless_checkout_happiness_link_clicked',
+																{
+																	product_slug: productSlug,
+																}
+															)
+														)
+													}
+													href={ happinessAppointmentLink }
+												/>
+											),
+										},
+									}
+								) }
+							</p>
+						</>
+					) }
+					{ automaticTransferSucceeded && (
+						<>
+							<p>
+								{ productName &&
+									rawSite?.name &&
+									translate(
+										"We successfully activated %(productName)s on {{strong}}%(siteName)s{{/strong}}. Next, we'll recommend features based on your goals",
+										{
+											args: { productName, siteName: rawSite.name },
```

Comment:
More often than not, the site name seems to be separated on 2 lines. Could we replace spaces by non breakable spaces? Definitely a nice-to-have.

Reply:
Good call. I'm definitely fixing this also.

Reply:
I'm late to the party here, but in case anyone's curious about how to make sure the site name's on one line, my first approach would probably be to change the value of the `siteName` arg to `preventWidows( rawSite.name )`. This will ensure that only site names with more than two "words" ever get wrapped.

Reply:
Ah -- I see Renzo accomplished it with a CSS rule, which works just as well! 🚀